OptimizelySDK.Tests/EventTests/CanonicalEvent.cs

```diff
@@ -0,0 +1,66 @@
+﻿using OptimizelySDK.Entity;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OptimizelySDK.Tests.EventTests
+{
+    public class CanonicalEvent
```

Comment:
Why are we calling this a `CanonicalEvent`?

Reply:
The name is same as in the Java SDK:
https://github.com/optimizely/java-sdk/blob/88d0e6a33ce26b30a01765c57e274c2f86d2dc17/core-api/src/test/java/com/optimizely/ab/EventHandlerRule.java#L161